Include/Library/OcCpuLib.h

```diff
@@ -104,6 +105,13 @@ typedef struct {
   //
   UINT64                  CPUFrequencyFromART;
 
+
+  //
+  // The CPU frequency derived from the CPUID VMWare Timing leaf.
+  // 0 if VMWare Timing leaf is not present.
+  //
+  UINT64                  CPUFrequencyFromVMWare;
```

Comment:
Does the variable name okay?

Reply:
I thought it is VMware, perhaps CPUFrequencyFromHypervisor is better, but I personally do not mind if it is the only change.